### PR TITLE
Adapt to new css selector for submitting OTP

### PR DESCRIPTION
**Pull request by tsdg112**
Mint has a new css selector for submitting a one time password.  This pull request works for either the old or new ways to submit the OTP.

### CODE REVIEW

**Comment by msimoni18** [comment]
I tested the changes in this PR by cloning the repo, ```python -m pip install -e .```, and then running the code I submitted in #364. It got further this time, but now there is some other error that popped up quickly on Mint saying something went wrong and took me back to the login screen.

**Comment by pazwicker** [comment]
I tested this too and found that we also need to account for a change to `mfa_code_input`:

```python
mfa_code_input = driver.find_element_by_id("iux-mfa-soft-token-verification-code")
```

I got this to work, but I can't for the life of me get the driver to click on "continue." I tried getting by CSS Selector, XPath, link text, but no dice.

File: mintapi/api.py [hunk 1]
```diff
@@ -474,9 +474,15 @@ def sign_in(
...
-                    mfa_code_submit = driver.find_element_by_id(
-                        "ius-mfa-otp-submit-btn"
-                    )
+                    try:
+                        mfa_code_submit = driver.find_element_by_id(
+                            "ius-mfa-otp-submit-btn"
+                        )
+                    except NoSuchElementException:
+                        mfa_code_submit = driver.find_element_by_css_selector(
+                            '[data-testid="VerifyOtpSubmitButton"]'
```
Comment (mrooney) [inline mintapi/api.py:483]: Does find_element_by_css_selector support commas? If so, could we simply do `driver.find_element_by_css_selector('#ius-mfa-otp-submit-btn, [data-testid="VerifyOtpSubmitButton"]')`

**Comment by tsdg112** [comment]
I didn't know that you could do that!  I made the change there and in another place in the code.

**Comment by mrooney** [comment]
I wasn't sure either, but if it is working on your end after the change, I guess we can :D

**Comment by tsdg112** [comment]
Yes, I tested it and its working, at least until Mint makes more changes on their end.

**Review by mrooney — APPROVED** [review]
Thanks, looks great to me!

**Comment by mrooney** [comment]
Created #372 to address the potential variable typo.

**Pull request merged**